README.md

```diff
@@ -18,11 +18,7 @@ We'll happily review any suggestions!
 git clone git@github.com:rkvst/rkvst-docs.git
 ```
 
-2. Move into the directory.
-
-```bash
-cd rkvst-doks
-```
+2. Ensure current directory is root of repo.
 
```

Comment:
I'd still leave an example of what the root directory should be here because not every user of the docs is a developer. 
```bash
cd rkvst-docs
```